### ISSUE TITLE
UI focus issue on Mac 26

### ISSUE THREAD

**Issue by saeedvaziry** [muxy-app/muxy#104]
On macOS 26, UI is forced to focus on terminal. For example opening the file picker doesn't have a focus on the file picker search or keyboard navigation doesnt work and keys apply to terminal under the modal.

Same happens when the theme picker is open, keys are applying to the terminal instead when the picker is open.
